src/test/resources/testconfigs/tr_demo1.txt

```diff
@@ -1,7 +1,7 @@
-keyboard=fr
-width=40px
-height=40px
-spacing=1px
+[keyboard = fr]
```

Comment:
We should introduce a sample where the keyboard is actually relevant. Surprised it's not here.